### PR TITLE
#61: API updates (closes #61)

### LINKED ISSUE #61: API updates
[Trello card](https://trello.com/c/58b958a6bfe88986795ae845/)

## requirements
The API version deployed on 8 March 2017 has changed a few things:

* `mobileDevices.type`: `["IOS", "GCM", "WP"]` -> `["IOS", "ANDROID", "WINDOWS_PHONE", "FIREOS"]`.
* `mobileDevices.appId`: new property (string)
* `mobileDevices.notificationService`: new property (`["APN", "GCM", "WNS", "ADM", "SNS"]`)

New events were also added:

* `eventConfirmed`
* `eventDeclined`
* `eventEligible`
* `eventNoShow`
* `eventNotInvited`

## specs
* update `types.js` (`EventType`, `MobileDevice` and `APIMobileDevice` types)
* update `customer.e2e.js`

## misc
ContactLab has deleted a few items from the test workspaces as part of the migration. This may cause some failing tests.

### PR DESCRIPTION
**Pull request by ascariandrea**
Closes #61

### tests performed
Happy CI

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.}

### CODE REVIEW

File: src/types.js [hunk 1]
```diff
@@ -276,9 +286,11 @@ export type Event = EventData & {
 // API Customer
 
 export type APIMobileDevice = {
+  appId: string,
   identifier: ?string,
   name: ?string,
-  type: null | 'IOS' | 'GCM' | 'WP'
+  type: null | MobileDeviceType,
```
Comment (dhinus) [inline src/types.js]: `?MobileDeviceType` should work

Comment (ascariandrea) [inline src/types.js]: Done!

File: src/types.js [hunk 2]
```diff
@@ -289,7 +289,7 @@ export type APIMobileDevice = {
   appId: string,
   identifier: ?string,
   name: ?string,
-  type: null | MobileDeviceType,
+  type: ?MobileDeviceType,
   notificationCenter: null | MobileDeviceNotificationCenter
```
Comment (dhinus) [inline src/types.js]: same thing on the line below :grin:

Comment (ascariandrea) [inline src/types.js]: 😎

**Comment by dhinus** [comment]
:shipit:

**Pull request merged**